static/package-esm.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
hmmm.. can you please explain what this file is for?

Reply:
The NPM script `build:copy-esm-packagejson` puts this file into the folder `./dist/esm`.  
It's sole purpose is that Node.js interprets `./dist/esm/tiny-invariant.js` as an ESM module.

Dr. Axel Rauschmayer wrote a blog post about "Hybrid npm packages", and Option 1 is the one we want to enable ("ESM and CommonJS are both bare imports"): <https://2ality.com/2019/10/hybrid-npm-packages.html#option-1-(experimental%2C-needs-conditional-exports)%3A-esm-and-commonjs-are-both-bare-imports>.  
Note that this blog post was last updated in 2019 and the things mentioned as experimental are not experimental anymore.